dev/io.openliberty.jaxws.security_fat.1/publish/files/JaxWsTransportSecurityServer/bindings/validNameButInvalidPwd.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you verify that all these binding files are needed by this test project, if their are binding files not being used by this particular project, can you delete the ones that aren't being used so we don't have any unnecessary duplicates?